### ISSUE TITLE
Macro Icons and Highlights

### ISSUE THREAD

**Issue by TheOSC** [brittyazel/Neuron#381]
When placing a macro on Neuron bars, on key press, the icon highlights but does not dehighlight unless you press the key again.

Also, I have a help harm macro for casting Stormstrike/Healing Surge, and another for casting Chain Lightning/Chain Heal. Regardless of my target state they remain on the icon for their respective heals and do not update ever. They do appear to show indications for proper range and the cool down for the proper skill, it is just the icon for the spell that does not update.

I have attempted /rl and exiting the game then re logging and neither issue are resolved with this.

**Comment by brittyazel** [comment]
Yeah I've encountered this as well. I'll try to get this sorted soon

**Comment by TheOSC** [comment]
Curious if there has been any progress on this? I am still experiencing issues with it. Not a huge deal but it does make the action bars feel a little janky as I use a lot of macros in general. I have also posted on issue [411](https://github.com/brittyazel/Neuron/issues/411) Which I feel is a major bug and it is affecting me quite a bit more than this. Any updates would be much appreciated.

I appreciate all the hard work you have put into Neuron and it is one of my favorite addons, I just want to see it become the best bar replacement around!

**Comment by ylixir** [comment]
i'm not sure what the macro highlight/unhighlight issue is? can you provide some screenshots?

if it is not still an issue, then we can repurpose this issue to deal with the mouseover updates
